srcalc/src/test/java/gov/va/med/srcalc/vista/RpcVistaPatientDaoTest.java

```diff
@@ -157,4 +160,36 @@ public final void testLabResponseNoUnits()
         // Insure the lab was not added and no exceptions occurred.
         assertEquals(1, patient.getLabs().size());
     }
+    
+    @Test
+    public final void testHealthFactorsValid()
+    {
+        final VistaProcedureCaller caller = mockVistaProcedureCaller();
+        when(caller.doRpc(
+                RADIOLOGIST_DUZ,
+                RemoteProcedure.GET_HEALTH_FACTORS,
+                String.valueOf(PATIENT_DFN)))
+                .thenReturn(Arrays.asList(VALID_HEALTH_FACTORS.split("\n")));
+        final RpcVistaPatientDao dao = new RpcVistaPatientDao(caller, RADIOLOGIST_DUZ);
+        final Patient patient = dao.getPatient(PATIENT_DFN);
+        assertEquals(3, patient.getHealthFactors().size());
```

Comment:
A clearer test would be to directly compare `patient.getHealthFactors()` with an expected list. See DerivedTermTest, line 61, for an example of what I mean.